lib/pipeline.js

```diff
@@ -469,6 +474,13 @@ class PipelineModel extends BaseModel {
                 // sort them by pr number
                 prJobs = prJobs.sort((job1, job2) => job1.prNum - job2.prNum);
 
+                if (config && config.type === 'pr') {
```

Comment:
I'd prefer if we use "pullrequest" instead of "pr"

Reply:
I was trying to keep it consistent with https://github.com/screwdriver-cd/models/blob/master/lib/eventFactory.js#L33 but I can change it if this is still better, what do you think?

Reply:
I'm okay with `pr`

Reply:
Ahh ok. I guess if it's already defined like that elsewhere, it's fine to keep it consistent here.